Source/DataSources/KmlTour.js

```diff
@@ -81,14 +82,42 @@ KmlTour.prototype.addPlaylistEntry = function (entry) {
   this.playlist.push(entry);
 };
 
+/**
+ * Play this tour.
+ *
+ * @param {CesiumWidget} widget The Cesium widget.
+ * @param {Object} [cameraOptions] these options will be merged with {@link Camera#flyTo}
+ * options for FlyTo playlist entries.
+ */
+KmlTour.prototype.playInWidget = function (widget, cameraOptions) {
+  this.tourStart.raiseEvent();
+
+  const tour = this;
+  playEntry.call(this, widget, cameraOptions, function (terminated) {
+    tour.playlistIndex = 0;
+    // Stop nonblocking entries
+    if (!terminated) {
+      cancelAllEntries(tour._activeEntries);
+    }
+    tour.tourEnd.raiseEvent(terminated);
+  });
+};
+
 /**
  * Play this tour.
  *
  * @param {Viewer} viewer viewer widget.
  * @param {Object} [cameraOptions] these options will be merged with {@link Camera#flyTo}
  * options for FlyTo playlist entries.
+ *
+ * @deprecated
  */
 KmlTour.prototype.play = function (viewer, cameraOptions) {
+  deprecationWarning(
+    "KmlTour.prototype.play",
```

Comment:
What's the motivation to deprecate this whole function? It's possible to deprecate just first parameter instead. [For example](https://github.com/CesiumGS/cesium/pull/6331/files#diff-d3495d81898f88370dc3635be83a0f6d87b7acf98c676debde79c59b852e907dR41).

Reply:
If we wanted to do just a parameter deprecation, we'd have to do an `instanceof` check against `Viewer` to print the `deprecationWarning`, right? Then we'd have to import `Viewer`, `which will cause issues in the workspaces PR.

Reply:
Good point. `instanceof` is generally what we'd use since it's a more cut and dry check, but in this case I agree we want to avoid importing the class. We could check for the presence of a `Viewer` property that is not present on an instance `CesiumWidget`, say `.cesiumWidget`?

Reply:
Yeah, that's a good idea! To ensure that we don't break TypeScript, I changed the types to `{Viewer|CesiumWidget}`. I'll create an issue to change that in 1.100. Then, in the workspaces PR, I can just add the following to make TS compile:

```typescript
/**
 * @typedef {Viewer} Viewer
 * @property {Scene} scene The scene in the widget.
 */
```